### DISCUSSION TITLE
User/employee activities not shown

### DISCUSSION THREAD

**Discussion by krseguros** [YetiForceCompany/YetiForceCRM#16683] [How to] [answered]
Hello,

I am the administrator user, however, I want to know why the User/employee Activities are not displayed in Company/Employees
Details.

If I can see Time Control logs but not calls, tasks, and meetings:

user screen
![image](https://user-images.githubusercontent.com/96985476/203444349-12f95fe8-4334-4a81-bd61-8776b7ba49bc.png)

administrator screen
![image](https://user-images.githubusercontent.com/96985476/203444511-623bedec-cbc7-436a-af88-0efb3fd05db0.png)

**Comment by rskrzypczak** [comment, marked as answer]
The answer is quite simple.
Both modules you indicate are on the same hierarchy level. In short, this means that a calendar entry cannot be related to a lead and an employee at the same time in the context of the relationship.
![image](https://user-images.githubusercontent.com/10221999/203473322-7a892a2b-a961-4c53-89fd-63fc6c90af9b.png)


On the other hand, the Time control tab on an employee works a little differently and is based on the record owner field.

**Comment by krseguros** [comment]
Understood, thank you.
